pkg/splunk/enterprise/configuration.go

```diff
@@ -606,3 +609,63 @@ func updateSplunkPodTemplateWithConfig(podTemplateSpec *corev1.PodTemplateSpec,
 		podTemplateSpec.Spec.Containers[idx].Env = env
 	}
 }
+
+// ListSmartStoreVolumes lists smartstore volumes
```

Comment:
I think "Log*" may be a better name for these vs more generic "List*", to help clarify the expected outcome